soccer/modeling/BallFilter.cpp

```diff
@@ -30,9 +30,34 @@ Ball BallFilter::predict(RJ::Time time, float* velocityUncertainty) const {
     const auto& vel = _currentEstimate.vel;
     const auto& pos = _currentEstimate.pos;
     const auto s0 = vel.mag();
-    auto part = std::exp(-0.2913f * t.count());
-    auto speed = s0 * part;
-    auto distance = s0 * -3.43289f * (part - 1.0f);
+
+//    const auto decayConstant = 0.1735;
+    const auto decayConstant = 0.1795;
+
+
+//    auto part = std::exp(-0.2913f * t.count());
```

Comment:
This part is copy pasted from https://github.com/RoboJackets/robocup-software/pull/911/files#diff-0050a07f81c01b7bc38ac572f8680d5dR88. 

Can you break it out into a function, perhaps?

Reply:
Whoops. Yeah I just used the Other ball filter.